ledger/src/transaction_address_lookup_table_scanner.rs

```diff
@@ -10,9 +9,8 @@ use {
     std::collections::HashSet,
 };
 
-lazy_static! {
-    static ref RESERVED_IDS_SET: HashSet<Pubkey> = ReservedAccountKeys::new_all_activated().active;
-}
+static RESERVED_IDS_SET: std::sync::LazyLock<HashSet<Pubkey>> =
+    std::sync::LazyLock::new(|| ReservedAccountKeys::new_all_activated().active.clone());
```

Comment:
Can we remove the `.clone()` ? If I'm not mistaken, we should be able to move `.active` instead of cloning a value that would immediately go out of scope